app/views/step_by_step_pages/show.html.erb

```diff
@@ -18,20 +18,27 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <% unless @step_by_step_page.can_be_published? %>
+    <% if !@step_by_step_page.can_be_published? && !@step_by_step_page.scheduled_for_publishing? %>
```

Comment:
Does the whole `inset` need to be wrapped in this? Surely, it only needs to be incorporated into the link check, as the step by step can't be published or scheduled if it doesn't have any steps etc.

Reply:
`can_be_published?` already does the `!@step_by_step_page.scheduled_for_publishing?` so it doesn't need to be repeated here.